cookbook/specs/spec_add_union.json

```diff
@@ -0,0 +1,99 @@
+{
+  "proposal": {
+    "title": "Add Specs: UNIONT",
+    "description": "Adding new specification support for relaying Union testnet data on Lava",
+    "specs": [
+      {
+          "index": "UNIONT",
+          "name": "union testnet",
+          "enabled": true,
+          "imports": [
+            "COSMOSSDK"
+          ],
+          "reliability_threshold": 268435455,
+          "data_reliability_enabled": true,
+          "block_distance_for_finalized_data": 0,
+          "blocks_in_finalization_proof": 1,
+          "average_block_time": 6180,
+          "allowed_block_lag_for_qos_sync": 2,
+          "shares": 1,
+          "min_stake_provider": {
+            "denom": "ulava",
+            "amount": "47500000000"
+          },
+          "api_collections": [
+            {
+              "enabled": true,
+              "collection_data": {
+                "api_interface": "rest",
+                "internal_path": "",
+                "type": "GET",
+                "add_on": ""
+              },
+              "apis": [],
+              "headers": [],
+              "inheritance_apis": [],
+              "parse_directives": [],
+              "verifications": [
```

Comment:
add pruning verification here and for the other api interfaces (total 3 verifications to add)

Reply:
Added them and commit it.